### PR TITLE
Landing page

### LINKED ISSUE #12: [feat]: create an initial project landing page
Hey, explorer! 🚀

Let's build a fancy landing page/website for the Astro Reactive Library. Does not have to be filled with content. The only condition is that it's built with Astro, because we love Astro! :)

Astro even offers [templates](https://astro.new) and [themes](https://astro.build/themes/)!

Acceptance criteria:
1. Start an Astro project called 'landing-page'
2. Choose a fancy theme that gives you a feeling of *wonder* and *exploration* 😉

See our [contributing guide](https://github.com/ayoayco/astro-reactive-library/blob/main/CONTRIBUTING.md) on how to run our project.

Don't hesitate to ask questions if something is not clear! :)

### PR DESCRIPTION
**Pull request by Preet-Sojitra**
Created basic landing page for the Astro Reactive Library . I haven't added any content. I used the theme listed on [astro/themes](https://astro.build/themes/)

Currently Landing page looks like this:

image

Fixes #12  

Description of changes: 
- Added landing page inside `landing-page/` directory

Tag a reviewer: @ayoayco 

Tasks:
- [x] I have ran the tests to make sure nothing is broken (see CONTRIBUTING.md)
- [x] I have ran the the linter to make sure code is clean (see CONTRIBUTING.md)

### CODE REVIEW

File: landing-page/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "astro-landing-page",
```
Comment (ayoayco) [inline landing-page/package.json:2]:
```suggestion
  "name": "astro-reactive-library-landing-page",
```

**Review by ayoayco — CHANGES_REQUESTED** [review]
Let's get this contribution in! 🚀 Just one change request :)

File: landing-page/package.json (hunk 1 above)
Comment (Preet-Sojitra) [reply landing-page/package.json:2]: Didn't noticed that 😅 . Will change it.

**Comment by Preet-Sojitra** [comment]
Requested changes have been made 👍

**Review by ayoayco — APPROVED** [review]
Nice work! 🚀

**Pull request merged**